apps/src/templates/rubrics/RubricFloatingActionButton.jsx

```diff
@@ -89,4 +92,11 @@ RubricFloatingActionButton.propTypes = {
   currentLevelName: PropTypes.string,
   reportingData: reportingDataShape,
   aiEnabled: PropTypes.bool,
+  sectionId: PropTypes.number,
 };
+
+export const UnconnectedRubricFloatingActionButton = RubricFloatingActionButton;
+
+export default connect(state => ({
+  sectionId: selectedSection(state)?.id,
+}))(RubricFloatingActionButton);
```

Comment:
Just so I know for the future- this is also where I would add any other states that are needed from redux, correct? Along with adding them to the import on line 15?

Reply:
yes, this is where you would pull any other data that's in redux into any rubric-related components. if the thing you need is in teacherSectionsRedux.js then the data you need may already be in redux. depending on what you need, you might need to do extra steps to put the data into redux. 

In this case I am actually having trouble finding out where the data is being put into redux, but i convinced myself that I could rely on it being there because it is already being depended upon by the teacher panel.

Reply:
Found it: https://github.com/code-dot-org/code-dot-org/blob/4fa2b9f0ebc62752f5cb13691b597c62172fc7e3/apps/src/code-studio/components/progress/teacherPanel/TeacherPanel.jsx#L104 this data is set in response to an ajax request once the page has already loaded, as opposed to being set at initial page load time.